chart/templates/pgbouncer/pgbouncer-deployment.yaml

```
@@ -66,6 +66,9 @@ spec:
       annotations:
         checksum/pgbouncer-config-secret: {{ include (print $.Template.BasePath "/secrets/pgbouncer-config-secret.yaml") . | sha256sum }}
         checksum/pgbouncer-certificates-secret: {{ include (print $.Template.BasePath "/secrets/pgbouncer-certificates-secret.yaml") . | sha256sum }}
+        {{- if .Values.pgbouncer.podAnnotations }}
+        {{- toYaml .Values.pgbouncer.podAnnotations | nindent 8 }}
```

```suggestion
          {{- toYaml .Values.pgbouncer.podAnnotations | nindent 8 }}
```

nit: since we can, let's indent this.

---

Done